src/agent/src/windows/windows_service.cpp

```diff
@@ -136,7 +136,7 @@ namespace windows_service
         CloseServiceHandle(schService);
         CloseServiceHandle(schSCManager);
 
-        LogInfo("Wazuh Agent Service successfully installed.");
+        LogInfo("wazuh-agent service successfully installed.");
```

Comment:
We should try to use the const string AGENT_SERVICENAME in the message, so the change is applied at a single point.